.circleci/config.yml

```diff
@@ -31,6 +31,7 @@ store-log-artifacts: &store-log-artifacts
 npm-cache-key: &npm-cache-key node-v10-{{ checksum "yarn.lock" }}-5
 bundle-cache-key: &bundle-cache-key v2-bundler-gems-{{ .Environment.DB }}-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
 assets-cache-key: &assets-cache-key v1-asset-cache-{{ checksum "tmp/assets_related_checksums" }}
+packs-cache-key: &packs-cache-key packs-cache-{{ checksum "tmp/packs_related_checksums" }}
```

Comment:
Good to add `v1-` prefix to the key. We can always add some prefix if we want to invalidate the key. Just for consistency with the others. Not mandatory.